BASE/Test/Microsoft.ApplicationInsights.Test/Microsoft.ApplicationInsights.Tests/Channel/TransmissionTest.cs

```diff
@@ -501,6 +501,31 @@ public async Task SendAsyncLogsIngestionReponseTimeAndStatusCode()
                     }
                 }
             }
+
+            [TestMethod]
+            public async Task TestTransmissionStatusEventHandler()
+            {
+                var handler = new HandlerForFakeHttpClient
+                {
+                    InnerHandler = new HttpClientHandler(),
+                    OnSendAsync = (req, cancellationToken) =>
+                    {
+                        return Task.FromResult<HttpResponseMessage>(new HttpResponseMessage());
```

Comment:
Additional test case: make this fail or throw exception or timeout.

Reply:
Done.